src/selectors/cashRegister.js

```diff
@@ -74,3 +75,12 @@ export const selectBalance = createSelector(
   [selectReceiptsTotal, selectPaymentsTotal],
   (receiptsTotal, paymentsTotal) => receiptsTotal.subtract(paymentsTotal).format()
 );
+
+export const selectCashRegisterInfoColumns = createSelector([selectBalance], currentBalance => [
+  [
+    {
+      title: `${pageInfoStrings.current_balance}:`,
+      info: currentBalance,
+    },
+  ],
+]);
```

Comment:
you reckon put all the page info sets in selectors? 

The current way makes a lot of the page info columns reuseable

Reply:
Just for a quick fix, this ended up being a bit quicker. I think they can be mixed though (still use the reusability we have, but derive the info columns from state using selectors). Not super worried, but its nice to shift as much to selectors as possible and the current way relies on storing a function in state, which I think we are trying to avoid?

Reply:
yeah you're right that isn't great !